File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestInsertionSort.java
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test;

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.InsertionSort;

public class TestInsertionSort extends TestSortingAlgorithm {
    @Override
    int[] sort(int[] randomNumbers) {
        return new InsertionSort().sortResult(randomNumbers).getSortedNumbers();
    }
}

File: test/tw/rmstudio/mt/java/course/mcu/ds/ex1/test/TestInsertionSort.kt
package tw.rmstudio.mt.java.course.mcu.ds.ex1.test

import tw.rmstudio.mt.java.course.mcu.ds.ex1.sorting.InsertionSort

class TestInsertionSort : TestSortingAlgorithm() {
    override fun sort(randomNumbers: IntArray) = InsertionSort().sortResult(randomNumbers).sortedNumbers
}
